rct229/rules/section22/section22rule20.py

```diff
@@ -0,0 +1,101 @@
+from rct229.data_fns.table_3_1_3_11_fns import table_3_1_3_11_lookup
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.schema.config import ureg
+from rct229.utils.assertions import assert_
+from rct229.utils.pint_utils import CalcQ
+from rct229.utils.std_comparisons import std_equal
+
+APPLICABLE_SYS_TYPES = [
+    "SYS-7",
+    "SYS-8",
+    "SYS-11.1",
+    "SYS-11.2",
+    "SYS-12",
+    "SYS-13",
+    "SYS-7B",
+    "SYS-8B",
+    "SYS-11B",
+    "SYS-12B",
+    "SYS-13B",
+]
+
+
+class Section22Rule20(RuleDefinitionListIndexedBase):
+    """Rule 20 of ASHRAE 90.1-2019 Appendix G Section 22 (Chilled water loop)"""
+
+    def __init__(self):
+        super(Section22Rule20, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            required_fields={
+                "$": ["weather"],
+                "weather": ["climate_zone"],
+            },
+            each_rule=Section22Rule20.HeatRejectionRule(),
+            index_rmr="baseline",
+            id="22-20",
+            description="The baseline minimum condenser water reset temperature is per Table G3.1.3.11.",
+            list_path="ruleset_model_instances[0].heat_rejections[*]",
+            # required_fields={
+            #     "$": ["ruleset_model_instances"],
```

Comment:
You can change line 32 to `"$": ["ruleset_model_instances", "weather"],`

Reply:
Fixed. Thanks!